src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -129,18 +132,19 @@ public void testWithMultipleChecks() throws Exception {
     }
 
     @Test
-    public void testAntlrError() throws Exception {
+    public void testAntlrError(final Capturable errStream) throws Exception {
```

Comment:
why `final` is required here?

Reply:
Sorry, copypasted from example as is. Removed.